.github/workflows/perfScaleNut.yml

```diff
@@ -0,0 +1,32 @@
+# runs some very-large-repo tests (for windows filesystem limits) and commits perf results for comparison
+name: perf-scale-nuts
+on:
+  workflow_call:
+    inputs:
+      os:
+        type: string
+        required: false
+        default: 'ubuntu-latest'
+
+jobs:
+  perf-scale-nuts:
+    runs-on: ${{ inputs.os }}
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: latest
+          cache: yarn
+      - run: npm install -g sfdx-cli --omit=dev
+      - run: yarn install
+      - run: yarn build
+      - run: yarn test:nuts:scale
+      - uses: salesforcecli/github-workflows/.github/actions/gitConfig@main
```

Comment:
Is `shell: bash` going to cause issues if you run this on Windows? ([source](https://github.com/salesforcecli/github-workflows/blob/main/.github/actions/gitConfig/action.yml#L7))

Reply:
it has not so far.  Maybe git config is simple/universal enough to be ok.